README.md

```diff
@@ -125,11 +125,33 @@ cross-compiling will work on an x86-64 Linux machine:
 * `rustup target add i686-unknown-linux-gnu`
 * `cargo build --target i686-unknown-linux-gnu`
 
-# Minimum Supported Rust Version
+# Maintenance Policies
 
-This crate requires Rust 1.56.1 at a minimum. 3.x releases of this crate supported an MSRV of 1.41.
+## Minimum Supported Rust Version (MSRV)
 
-In the future, MSRV changes will be accompanied by a minor version bump.
+| Releases | MSRV   |
+| :---     | :---   |
+| 4.x      | 1.56.1 |
+| 3.x      | 1.41.0 |
+
+From 4.x MSRV changes will be accompanied by a minor version bump.
+
+## Public API SemVer Exemptions
+
+Breaking changes on SemVer exempted components affecting the Public API will also be accompanied by;
+
+| Releases | Public API Component(s)               | Policy              |
+| :---     | :---                                  | :---                |
+| 4.x      | Dependencies `digest` and `rand_core` | Minor SemVer bump   |
+
+* In the future exempted `digest` and `rand_core` may be changed to optional features.
+* Any changes to these exemptions will be accompanied by a major version bump.
```

Comment:
I think it would be OK to stabilize the associated version of `digest` and/or `rand_core` if they drop a 1.0 release without bumping `curve25519-dalek` to 5.0, e.g. hypothetically "all versions of `curve25519-dalek` v4.3.0 and newer use `rand_core` v1"

Reply:
Ok. Simplified this too.